rules-reviewed/eap6/websphere/websphere-catchall.windup.xml

```diff
@@ -17,6 +17,31 @@
         <tag>reviewed-2015-06-17</tag>
     </metadata>
     <rules>
+        <rule id="websphere-catchall-db2-00000">
+            <when>
+                <javaclass references="com.ibm.db2.jcc.{type}" />
+            </when>
+            <perform>
+                <iteration>
+                    <when>
+                        <not>
+                            <has-hint />
+                        </not>
+                    </when>
+                    <perform>
+                        <hint title="IBM DB2 driver type reference" effort="1" severity="potential">
```

Comment:
While I certainly agree with this, this particular case makes sense to have effort as that means you need to configure JNDI/Datasource.